data/categories.txt

```diff
@@ -9777,6 +9777,9 @@ shop-second_hand
 en:4Second Hand
 ru:4Подержанные товары
 
+shop-charity
+en:4Charity Shop
```

Comment:
Please translate using tools/unix/translate_categories.sh and then fix by using DeepL.

Reply:
Ah! That's how you get so many translations! Working on it :)